tests/e2e/expected_results/micronaut-test/micronaut-test.json

```diff
@@ -1,14 +1,14 @@
 {
     "metadata": {
-        "timestamps": "2023-11-09 12:01:32",
+        "timestamps": "2023-12-22 01:32:17",
         "has_passing_check": true
     },
     "target": {
         "info": {
             "full_name": "pkg:github.com/micronaut-projects/micronaut-test@7679d10b4073a3b842b6c56877c35fa8cd10acff",
             "local_cloned_path": "git_repos/github_com/micronaut-projects/micronaut-test",
             "remote_path": "https://github.com/micronaut-projects/micronaut-test",
-            "branch": "master",
+            "branch": null,
```

Comment:
Can you please leave at least one of the tests that uses both branch and commit.

Reply:
I think we still have a test with both branch and commit here - https://github.com/oracle/macaron/blob/c9f5a77f0ed7160fe21d34c9e26076e352ca2001/scripts/dev_scripts/integration_tests.sh#L223-L232.
Do you mean having a test case having both branch and commit defined in the yaml config file Behnaz ?

Reply:
yes

Reply:
Added a new test case in https://github.com/oracle/macaron/pull/548/commits/2f53835a7db412c10191caa22e806bbf54727a68

Reply:
I didn't mean to add a new test, but to keep the branch and commit in one of the existing tests. Adding a new test will make the CI runs longer. So, if not necessary I'd like to avoid that.

Reply:
We currently have this test case [here](https://github.com/oracle/macaron/blob/2f53835a7db412c10191caa22e806bbf54727a68/scripts/dev_scripts/integration_tests.sh#L302-L319) that use both branch and digest in the configuration file. We could still have it and I can revert the new test case I just added.